eng/mgmt/automation/generate.py

```diff
@@ -134,6 +134,8 @@ def update_changelog(changelog_file, changelog):
 
     first_version_part = old_changelog[:first_version.end() +
                                        second_version.start()]
+    # remove text starting from the first '###' (usually the block '### Features Added')
```

Comment:
@weidongxu-microsoft Do you guys use the auto-increment PRs that add these? @chidozieononiwu just added some logic that can be used to clean-up the empty sections (see https://github.com/Azure/azure-sdk-tools/pull/1979), however that still seems like it it might be unnecessary for you guys. We should consider adding some sort of opt-out if these headers aren't interesting for your generated change logs.

I will add however that even for the generated changelogs you should try using the consistent headers.

Reply:
I should also add we already have some PS helpers https://github.com/Azure/azure-sdk-for-java/blob/main/eng/common/scripts/ChangeLog-Operations.ps1 that handle parsing and writing of changelogs following our standard formatting and it would be good if you guys could try using those instead of having to maintain a separate parsing/writing library.

Reply:
@weshaggard 
For most mgmt SDKs, we relay on automated tool to fill the changelog section (from tool introspecting the API of previous lib and current lib). It follows the guideline and standard formatting.
Example:
https://github.com/Azure/azure-sdk-for-java/pull/24034/files

Here the change is just to clean-up the unused suggestion (that automatically filled when incremental version).

We will check the tool at https://github.com/Azure/azure-sdk-tools/pull/1979, but since anyway all the content in changelog is automated (but still manually reviewed), we are probably fine with the current Python script.

Reply:
Thanks for the information and for using the new headings. 

Please do check out the existing changelog scripts to see if makes since for you guys to use them and potentially add to them if needed. The main reason is to try and cut down on the cost it takes to maintain multiple sets of tools/scripts that do duplicated stuff. For example every time we make a change to our common processes you will also need to make a corresponding change like this one to fix up your scripts, so it is generally better to share instead of re-implement.